start.sh

```diff
@@ -3,6 +3,8 @@
 # Chown volumes so that nginx can read/write
 chown www-data:www-data /usr/share/nginx/html/images
 chown www-data:www-data /usr/share/nginx/html/LocalSettings.php
+chown -R www-data:www-data /tmp/extensions/
+cp -r /tmp/extensions/* /usr/share/nginx/html/extensions/
```

Comment:
Isn't it a bit confusing to the user who assume everything should be setup correctly without any differences in configuration? I meant, typical user just follow everything written in Wikimedia documentation website and copy-paste the sample configuration. For such users, they expect the ``require_once``  to refers to the same path as the one written in Wikimedia documentation (i.e. without ``docker/`` parent folder).